package.json

```diff
@@ -854,7 +860,7 @@
     "promisify-child-process": "4.1.1",
     "vscode-languageclient": "7.0.0"
   },
-  "extensionDependencies": [
+  "extensionPack": [
```

Comment:
I'm really glad you asked because I could have forgotten about one crucial thing. 

`extensionDependencies` means that it will be installed if the main extension is installed (since it depends on its API), I think `extensionPack` doesn't have this kind of guarantee. In this particular case, I changed it because the extension's tests were failing that there is no `scala-lang.scala` extension.

@tgodzik has a right, I'll do it in separate PR and make sure after that if installing Metals installs Scala Syntax too

Reply:
Actually I was wrong
<img width="767" alt="Screenshot 2022-01-20 at 13 06 34" src="https://user-images.githubusercontent.com/37124721/150335925-5522f6eb-94e5-4fbf-977e-1d2b4a37800a.png">